src/gui/generalsettings.ui

```diff
@@ -48,6 +48,53 @@
     </widget>
    </item>
    <item row="2" column="0">
+    <widget class="QGroupBox" name="advancedGroupBox">
```

Comment:
I still think there should be no UI addition.

Reply:
Then you can feel free to remove it before the merge.